templates/components/form/header.html.twig

```diff
@@ -135,6 +135,7 @@
                            {% set disabled = true %}
                         {% endif %}
 
+                        <input type="hidden" name="is_recursive" value="0" />
```

Comment:
When field is disabled, it is sometime ue to the fact that is_recursive=1 and should not be set to 0. I am not sure the 0 value should be sent in this case.